packages/server/test/support/fixtures/projects/studio/cypress/integration/events.spec.js

```diff
@@ -0,0 +1,71 @@
+import { openStudio, saveStudio, verifyCommandLog } from '../support'
```

Comment:
I think this spec would be a better fit as an [isolated runner spec](https://github.com/cypress-io/cypress/tree/develop/packages/runner/cypress/integration). It's easier to interact with and test the reporter/runner UI itself in those tests. The e2e snapshot here doesn't really provide any value, so we wouldn't be losing anything by getting rid of it.

In addition, it would be great to have a truly end-to-end test that interacts with Cypress Studio and then verifies that it writes the tests out to an actual file. I think that could potentially be an isolated runner test too, or a server e2e test if necessary.

Reply:
@chrisbreiding yeah that was the goal here, I was having trouble figuring out how to test the file writing in a full e2e tetst so I wanted to take another stab at that today - if I can't figure it out I can transition this to somewhere else like isolated runner